### DISCUSSION TITLE
change basis of pr-compliance-action

### DISCUSSION THREAD

**Discussion by mtfoley** [open-sauced/open-sauced#1268] [General] [answered]
As found on a recent PR https://github.com/open-sauced/open-sauced/pull/1245 , the pr-compliance-action had a bug where it didn't show as a failed check (but did raise an error) on a technically invalid title. The title was adjusted just to normalize everything, and I've cut a new release on pr-compliance-action to fix the behavior.

This was brought up in issue #1266 

However, what this brings to light is that our existing use of the action [amannn/action-semantic-pull-request](https://github.com/amannn/action-semantic-pull-request) is based on a different parser than that of pr-compliance-action

| action | parser |
| --- | --- |
| amannn/action-semantic-pull-request | `conventional-changelog-conventionalcommits` |
| mtfoley/pr-compliance-action | `@commitlint/config-conventional` |

From what I can see, `conventional-changelog-conventionalcommits` is less strict in that it allows "sentence" case and other cases for the "subject" portion of the commit message. 

So what I'm proposing is that I try to change the basis of pr-compliance-action to `conventional-changelog-conventionalcommits` and make sure tests still pass.

**Comment by 0-vortex** [comment, marked as answer]
That would be great because we are using it in our release configuration as well:

https://github.com/open-sauced/semantic-release-conventional-config/blob/748c9baa12dc4722c53b5ba16cab0c84fb4b148c/package.json#L29

**Reply by mtfoley** [reply]
Seals the deal for me - will let y’all know when I’ve got it working :-)

**Reply by mtfoley** [reply]
Supported in v0.2.0
